components/tools/OmeroPy/src/omero/plugins/ldap.py

```diff
@@ -102,6 +109,7 @@ def active(self, args):
         else:
             self.ctx.die(1, "No")
 
+    @windows_warning
```

Comment:
I'm not completely sure these LDAP ones are necessary. Here `admin_only` differs slightly from the `admin.py` plugin. The former means "member of system group in OMERO" whereas the second means "system administrator with access to the installation directories".